packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -32,11 +37,15 @@ export default class Atomic extends Command {
   @Trackable()
   @Preconditions(
     IsAuthenticated(),
+    IsNpxInstalled(),
+    IsNodeVersionInRange(Atomic.requiredNodeVersion),
     HasNecessaryCoveoPrivileges(createApiKeyPrivilege, impersonatePrivilege)
   )
   public async run() {
     await this.createProject();
     this.displayFeedbackAfterSuccess();
+    // TODO: add env variables
+    await this.config.runHook('analytics', buildAnalyticsSuccessHook(this, {}));
```

Comment:
We no longer need to call the `runHook` method. Analytic tracking is being tacking care of by the `@Trackable` decorator.